index.js

```diff
@@ -533,6 +535,10 @@ function Argv (processArgs, cwd) {
     })
   }
 
+  function guessLocale () {
+    return osLocale.sync().split('_')[0]
```

Comment:
@bcoe Created [y18n PR 4](https://github.com/bcoe/y18n/pull/4), though I'm not really in love with it.